src/DigitalProductPassport/MetaArtifact/DataSheet_v0.1.py

```diff
@@ -0,0 +1,116 @@
+from typing import List, Optional
+
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class EnStandardCertification(CamelCaseModel):
+    en_standard_code: Optional[str] = Field(
+        None,
+        title="EN Standard Code",
+        max_length=20,
+        description="The identification code of the EN standard that the product complies with",
+        example="EN 10002-1, EN 10002-5",
+    )
+
+
+class Measures(CamelCaseModel):
+    width: Optional[int] = Field(
+        None,
+        title="Width",
+        description="The width of the product measured in centimeters",
+        example=1500,
+    )
+    length: Optional[int] = Field(
+        None,
+        title="Length",
+        description="The length of the product measured in centimeters",
+        example=4000,
+    )
+    height: Optional[int] = Field(
+        None,
+        title="Height",
+        description="The height of the product measured in centimeters",
+        example=0,
+    )
+
+
+class MetaArtifactDataSheetRequest(CamelCaseModel):
+    product: str = Field(
+        ...,
+        title="Product identifier",
+        description="Technical product identifier used by the manufacturer",
+        example="battery-100wh-s",
+    )
+    id: str = Field(
+        ...,
+        title="Identifier",
+        description="Unique identifier for the product",
+        example="177389-09633",
+    )
```

Comment:
This data was a bit changed in the sheet, I think it should now be like this (check before applying I've not made any mistake).

```suggestion
    product: str = Field(
        ...,
        title="Product code",
        description="The product code used for identifying the product type",
        example="steel-roll-1234x",
    )
    id: str = Field(
        ...,
        title="Identifier",
        description="Unique identifier of the product",
        example="550e8400-e29b-41d4-a716-446655440000",
    )
```